project3/styles.css

```diff
@@ -0,0 +1,285 @@
+* {
+    box-sizing: border-box;
+    transition: 400ms;
+}
+
+
+body {
```

Comment:
Remember to also include base rules for `font-family` and text `color` on your `body` element.